transform/convert_quadkey_and_Vertical_id.go

```diff
@@ -342,6 +342,86 @@ func ConvertExtendedSpatialIDsToQuadkeysAndVerticalIDs(extendedSpatialIDs []stri
 	return extendedSpatialIDToQuadkeyAndVerticalID, nil
 }
 
+func ConvertExtendedSpatialIDsToQuadkeysAndAltitudekeys(extendedSpatialIDs []string, outputHZoom int64, outputVZoom int64, altitudeRangeScalar int64, verticalIndexOffset int64) ([]*object.FromExtendedSpatialIDToQuadkeyAndAltitudekey, error) {
+	extendedSpatialIDToQuadkeyAndAltitudekey := []*object.FromExtendedSpatialIDToQuadkeyAndAltitudekey{}
+
+	// validate zoom levels
+	if !quadkeyCheckZoom(outputHZoom, outputVZoom) {
+		return []*object.FromExtendedSpatialIDToQuadkeyAndAltitudekey{}, errors.NewSpatialIdError(errors.InputValueErrorCode, "")
+	}
+	// outputのZoomレベルが指定されている前提のため、QuadkeyとAltitudekeyのみを比較
+	duplicate := map[[2]int64]interface{}{}
```

Comment:
I am sorry, there is a word [deduplication](https://ejje.weblio.jp/content/deduplication).
Please revert it.